test_fms/mpp/test_domains_utility_mod.F90

```diff
@@ -71,12 +72,12 @@ subroutine fill_coarse_data_r8(data, rotate, iadd, jadd, is_c, ie_c, js_c, je_c,
        do k = 1, nz
           do j = js_c, je_c+jshift
              do i = is_c, ie_c+ishift
-                data(i,j,k) = sign2*((j+iadd)*1.e+6 + (ny-i+1+jadd+ishift)*1.e+3 + k + y_add)
+                rdata(i,j,k) = sign2*( dble(j+iadd)*1.d+6 + dble(ny-i+1+jadd+ishift)*1.d+3 + dble(k) + y_add)
              enddo
           enddo
        enddo
     case default
-       call mpp_error(FATAL,"fill_coarse_data: rotate_coarse must be ZERO, NINETY, MINUS_NINETY")
+       call mpp_error(FATAL,"fill_coarse_rdata: rotate_coarse must be ZERO, NINETY, MINUS_NINETY")
```

Comment:
Should this be "fill_coarse_data_r8"?

Reply:
Yes fixed this and the r4 message as well